agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -93,14 +93,26 @@ while true; do
 		;;
 	esac
 done
+
+function setup_ara {
```

Comment:
ara needs to be setup before a benchmark run starts so that we can collect the data once the benchmark run completes.